app/_src/gateway/kong-enterprise/signing-verify/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Verify signatures for Signed Kong Images
+badge: enterprise
+---
+
+Starting with {{site.ee_product_name}} 3.5.0.2, Docker container images are now signed using `cosign` with signatures published to a Docker Hub repository.
+
+This guide provides steps to verify signatures for signed {{site.ee_product_name}} Docker container images in 2 different ways. The first is the minimal example, used to verify an image without leveraging any annotations. The second is the complete example, leveraging optional annotations for increased trust.
+
+For the minimal example, you only need: Docker details, Github repo name, and Github workflow filename.
+For the complete example, you will need the same details from above, as well as any of the optional annotations you wish to verify:
+
+| Shorthand | Description | Example Value |
+|---|---|---|
+| `<repo>` | Github repository | `kong-ee` |
+| `<workflow filename>` | Github workflow filename | `release.yml` |
+| `<workflow name>` | Github workflow name | `Package & Release` |
+
+Because Kong uses Github Actions to build and release, Kong also uses Github's OIDC identity to sign images; thus many of these details are Github-related.
```

Comment:
```suggestion
Because Kong uses Github Actions to build and release, Kong also uses Github's OIDC identity to sign images, which is why many of these details are Github-related.
```